src/interface/src/app/services/plan-state.service.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
+
+import { PlanStateService } from './plan-state.service';
+import { PlanService } from './plan.service';
+import { ScenarioService } from './scenario.service';
+import { TreatmentGoalsService } from './treatment-goals.service';
+import { of } from 'rxjs';
+
+describe('PlanStateService', () => {
+  let service: PlanStateService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        { provide: PlanService, useValue: {} },
+        { provide: ScenarioService, useValue: {} },
+        {
+          provide: TreatmentGoalsService,
+          useValue: { getTreatmentGoalsForArea: () => of([]) },
+        },
+      ],
+    });
+    service = TestBed.inject(PlanStateService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
Did some of the relevant tests get folded into another spec file? Apologies if I'm somehow not seeing them.

Reply:
There aren't any for those methods :/ I moved the ones we have for scenario to the scenario spec file but I dont think we have tests for any of this stuff

Reply:
ok, no worries